docs/guides/formulasC2/formula-instance.md

```diff
@@ -0,0 +1,92 @@
+---
+heading: Building Formulas
+seo: Build a Formula Template | Formulas | Cloud Elements API Docs
+title: Create a Formula Instance
+description: How to build a formula template
+layout: sidebarleft
+apis: API Docs
+platform: formulas
+breadcrumbs: /docs/guides/home.html
+parent: Back to Guides
+order: 10
+---
+
+# Formula Instances
+
+Formula instances are individual copies of a formula template with specific element instances and values replacing the Element Instance and Value variables in the template. You can set up formula instances to provide notifications of any errors through email for a webhook URL.
```

Comment:
I think instance is the best word, personally.  I don't know if occurrences makes sense either?  🤔